Fatras/include/ActsFatras/EventData/Particle.hpp

```diff
@@ -152,6 +152,10 @@ class Particle {
   constexpr ProcessType process() const { return m_process; }
   /// PDG particle number that identifies the type.
   constexpr Acts::PdgParticle pdg() const { return m_pdg; }
+  /// Absolute PDG particle number.
+  constexpr Acts::PdgParticle absolutePdg() const {
```

Comment:
`absolutePdg` vs `absPdg`

Reply:
below we have `absoluteCharge` that is why I was a bit conflicted with `absPdg` :smile: